lib/attributes/utils/common.cpp

```diff
@@ -0,0 +1,20 @@
+#include "attributes/utils/common.h"
+
+namespace oklt {
+bool isLastOuter(OklLoopInfo* loop) {
```

Comment:
It is better to be a part of `OklLoopInfo`

Reply:
Done